### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by rubenderigo**
Add initial footer

### CODE REVIEW

File: src/features/common/layouts/MainLayout/MainLayout.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import PropTypes from 'prop-types';
 
 import { Header, ShapesBackground } from 'features/common';
+import Footer from 'features/common/components/Footer/Footer';
```
Comment (alejandroGonGon) [inline src/features/common/layouts/MainLayout/MainLayout.js]: 👀

File: src/layouts/MainLayout.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
 import PropTypes from 'prop-types';
 
 import { Header, ShapesBackground } from 'features/common';
+import Footer from 'features/common/components/Footer/Footer';
```
Comment (alejandroGonGon) [inline src/layouts/MainLayout.js]: 🙄

**Review by alejandroGonGon — CHANGES_REQUESTED** [review]
Change the Footer to Index (imports)

File: src/features/common/components/Footer/Footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,100 @@
+@import 'sass/variables';
+
+.footer {
+  background-color: #353434;
+  color: #bcbcbc;
```
Comment (nicocadq) [inline src/features/common/components/Footer/Footer.module.scss]: This colors should come from the variables

File: src/features/common/components/Footer/Footer.module.scss [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  font-size: $font-small;
+  height: fit-content;
+  padding: 2rem;
+  width: 100%;
+  z-index: 999;
```
Comment (nicocadq) [inline src/features/common/components/Footer/Footer.module.scss:10]: Should we add indexes to the variables?

File: src/features/common/components/Footer/Footer.module.scss [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+.contact__us,
+.powered {
+  display: flex;
+  align-items: center;
+
+  svg {
+    width: 35px;
+    height: 35px;
+    margin: 5px;
```
Comment (nicocadq) [inline src/features/common/components/Footer/Footer.module.scss]: We are using rem in this project

File: src/features/common/components/Footer/Footer.module.scss [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    margin-bottom: 0;
+  }
+}
+
+.logo {
+  align-items: center;
+  display: flex;
+  color: $white;
+  font-size: $font-x-small;
+  margin-bottom: 7px;
```
Comment (nicocadq) [inline src/features/common/components/Footer/Footer.module.scss]: rem here

File: src/features/common/components/Footer/Footer.module.scss [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+  svg path {
+    fill: $white;
+  }
+
+  svg {
+    width: 25px;
+    height: 25px;
```
Comment (nicocadq) [inline src/features/common/components/Footer/Footer.module.scss]: same about rems

File: src/features/common/components/Footer/Footer.module.scss [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+  }
+}
+
+.contact__us {
+  align-items: center;
+
+  .contact__us__title {
+    font-size: $font-small;
+    margin-right: 6px;
```
Comment (nicocadq) [inline src/features/common/components/Footer/Footer.module.scss]: 👀

File: src/features/common/components/Footer/Footer.module.scss [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+  font-size: $font-x-small;
+
+  a {
+    display: flex;
+    align-items: center;
+  }
+
+  svg {
+    width: 20px;
+    height: 20px;
```
Comment (nicocadq) [inline src/features/common/components/Footer/Footer.module.scss]: 👀

File: src/features/common/components/Footer/Footer.module.scss [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+  .container {
+    flex-direction: column;
+  }
+
+  .copyright,
+  .contact__us,
+  .powered,
+  .logo {
+    display: flex;
+    margin: 20px 0;
```
Comment (nicocadq) [inline src/features/common/components/Footer/Footer.module.scss]: Same here

File: src/features/common/components/Footer/Footer.module.scss [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+    justify-content: center;
+    text-align: center;
+  }
+
+  .logo {
+    margin-bottom: 2px;
```
Comment (nicocadq) [inline src/features/common/components/Footer/Footer.module.scss]: 👀

File: src/features/common/components/Header/Header.module.scss [hunk 12]
```diff
@@ -1,27 +1,26 @@
 @import 'sass/variables';
 
 .header {
-  height: 80px;
-  width: 100%;
+  align-items: center;
+  background-color: $white;
   box-shadow: 0px 4px 11px 0px #73b8b233;
```
Comment (nicocadq) [inline src/features/common/components/Header/Header.module.scss:6]: I think that this should come from shadow variables

File: src/features/common/components/Header/Header.module.scss [hunk 13]
```diff
@@ -33,29 +32,29 @@
...
-    width: 400px;
-  }
-
-  .nav_donation-request {
-    width: 250px;
+    align-items: center;
+    display: flex;
+    gap: 1rem;
+    margin-right: 5rem;
+    max-width: 250px;
```
Comment (nicocadq) [inline src/features/common/components/Header/Header.module.scss:44]: rem

File: src/features/common/components/Header/Header.module.scss [hunk 14]
```diff
@@ -77,14 +76,14 @@ button {
 }
 
 .nav__dropdown {
-  position: absolute;
   background-color: $white;
   border-radius: 14px;
```
Comment (nicocadq) [inline src/features/common/components/Header/Header.module.scss:80]: add this to radius constants

File: src/features/common/components/Header/Header.module.scss [hunk 15]
```diff
@@ -170,17 +169,19 @@ button {
 
 @media only screen and (max-width: 768px) {
   .header {
-    @apply p-4 lg:p-7 bg-white;
+    @apply lg:p-7 bg-white;
```
Comment (nicocadq) [inline src/features/common/components/Header/Header.module.scss:172]: maybe we could use our own css here

File: src/features/common/components/Header/Header.module.scss [hunk 16]
```diff
@@ -170,17 +169,19 @@ button {
...
 
     box-shadow: 0px 4px 11px 0px #73b8b233;
```
Comment (nicocadq) [inline src/features/common/components/Header/Header.module.scss:174]: this should come from constatns

File: src/features/common/components/NotFound/NotFound.module.scss [hunk 17]
```diff
@@ -1,8 +1,9 @@
 @import '../../../../sass/variables';
```
Comment (nicocadq) [inline src/features/common/components/NotFound/NotFound.module.scss]:
```suggestion
@import 'sass/variables';
```

File: src/features/common/layouts/Main/MainLayout.js [hunk 18]
```diff
@@ -1,13 +1,16 @@
...
+import { Footer, Header, ShapesBackground } from 'features/common';
 
 const MainLayout = ({ children }) => (
-  <div className="w-full mx-auto min-h-screen min-w-full">
+  <>
     <ShapesBackground />
     <Header />
-    <div className="my-8 max-w-screen-lg mx-auto relative">{children}</div>
-  </div>
+    <main className="my-8 max-w-screen-lg mx-auto w-full relative min-h-screen">
```
Comment (nicocadq) [inline src/features/common/layouts/Main/MainLayout.js]: We should we add this into our own css

File: src/index.css [hunk 19]
```diff
@@ -2,6 +2,21 @@
 @tailwind components;
 @tailwind utilities;
 
+
 #root {
-  @apply bg-white min-h-screen w-full flex justify-center items-center;
+  @apply bg-white w-full;
```
Comment (nicocadq) [inline src/index.css]: we could use our own css here

File: src/pages/NotMatch/NotMatch.js [hunk 20]
```diff
@@ -1,7 +1,7 @@
 import { ComeBackButton, NotFound } from 'features/common';
 
 const NotMatch = () => (
-  <div>
+  <div className="m-auto">
```
Comment (nicocadq) [inline src/pages/NotMatch/NotMatch.js]: 👀

File: src/sass/_variables.scss [hunk 21]
```diff
@@ -32,3 +32,14 @@ $breakpoint-mobile-small: 425px;
...
+
+// Radius
+$radius-full: 9999px;
+
+// Fonts
+$font-x-small: 12px;
+$font-small: 14px;
+$font-medium: 16px;
+$font-large: 18px;
+$font-x-large: 20px;
```
Comment (nicocadq) [inline src/sass/_variables.scss]: Should we add this in rem?

**Review by nicocadq — APPROVED** [review]

File: src/features/common/layouts/Main/MainLayout.module.scss [hunk 22]
```diff
@@ -0,0 +1,9 @@
+//"my-8 max-w-screen-lg mx-auto w-full relative min-h-screen"
+
+.main {
+  margin: 2rem auto;
+  max-width: 64rem;
```
Comment (nicocadq) [inline src/features/common/layouts/Main/MainLayout.module.scss:5]: This should be into a variable

File: src/sass/_variables.scss [hunk 23]
```diff
@@ -20,15 +20,28 @@ $email: #d24538;
...
+$breakpoint-mobile-small: 26.56rem;
 
 // Utilities
-$header-height: 80px;
-$max-width: 1080px;
+$header-height: 5rem;
+$max-width: 67.5rem;
+
+// Radius
+$radius-full: 624.93rem;
```
Comment (nicocadq) [inline src/sass/_variables.scss:39]: I think that this values should be defined on px

**Review by nicocadq — APPROVED** [review]
just try to take a look at the comments 🤙🏻

**Pull request merged**